src/avfoundation.cs

```diff
@@ -8062,6 +8278,31 @@ interface AVComposition_AVCompositionTrackInspection {
 		void LoadTracksWithMediaCharacteristic (string mediaCharacteristic, Action<NSArray<AVMutableCompositionTrack>, NSError> completionHandler);
 	}
 
+	[Watch (9,0), TV (16,0), Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
+	[Category]
+	[BaseType (typeof(AVComposition))]
+	interface AVComposition_SynchronousAssetInterface
+	{
+		[Export ("metadataForFormat:")]
+		AVMetadataItem[] GetMetadataForFormat (string format);
```

Comment:
Do we want this to be Internal?
```suggestion
		AVMetadataItem[] GetMetadataForFormat (NSString format);
```